content/project/utilities/rubicon/contents.lr

```diff
@@ -1,29 +1,26 @@
 name: Rubicon
 ---
-project_type: bridge
+project_type: utility
 ---
 maturity: production
 ---
-languages: py, m, java
+languages: py, m
 ---
 platforms: macOS, iOS
 ---
 short_description: A library for bridging between Python and other language runtimes.
 ---
 description:
 
-Rubicon is a namespace holder for multiple different parsers:
+Rubicon is a bridge interface between Python and Objective-C.
```

Comment:
We can beef this up with the text from the Rubicon ObjC docs.
```suggestion
Rubicon Objective-C is a bridge between Objective-C and Python. It enables you to:

* Use Python to instantiate objects defined in Objective-C,
* Use Python to invoke methods on objects defined in Objective-C, and
* Subclass and extend Objective-C classes in Python.

It also includes wrappers of the some key data types from the Foundation framework (e.g., `NSString`).
```

Reply:
This page has been updated with all of the above suggestions.